packages/esm-patient-common-lib/src/lab-resource/types/order.ts

```diff
@@ -0,0 +1,16 @@
+import { type Order } from '@openmrs/esm-patient-common-lib';
+
+export interface PatientMedicationFetchResponse {
+  results: Array<Order>;
+}
+
+export interface OrderDiscontinuationPayload {
+  previousOrder: string;
+  type: string;
+  action: string;
```

Comment:
I feel like order action is a restricted (enum) type we already have defined. Please re-use that.